docs/strategies/circuit-breaker.md

```diff
@@ -203,6 +203,8 @@ Let's suppose we have a circuit breaker strategy with the following configuratio
 
 #### Complex: happy path sequence diagram
 
+The circuit will break and later it will transition into `HalfOpen` state. The probe will succeed so, the circuit breaker will go back to normal (`Closed`) state.
```

Comment:
```suggestion
The circuit will break and later it will transition into the `HalfOpen` state. The probe will then succeed, so the circuit breaker will go back to the normal (`Closed`) state.
```